.github/workflows/deploy-person-api.dev-u.yml

```diff
@@ -0,0 +1,49 @@
+name: Deploy person-api u2
```

Comment:
Jeg ville klat denne filen for bare deploy-person-api.dev.yml

Reply:
Var usikker på den ja, men deploy-person-api.dev.yml eksisterer allerede for t2 :/ Er det mulig å merge de to filene siden de nærmest er identiske?

Reply:
Ja skal gjøre dette men bare for t2 u2:
https://github.com/navikt/testnorge/blob/master/.github/workflows/deploy-testnorge-aareg.yml